lib/dynflow/exporters/tar.rb

```diff
@@ -0,0 +1,81 @@
+require 'zlib'
+require 'archive/tar/minitar'
```

Comment:
Would we try to use http://ruby-doc.org/stdlib-2.2.2/libdoc/rubygems/rdoc/Gem/Package/TarWriter.html, to minimize external dependencies?